detok/decode.c

```diff
@@ -549,28 +544,26 @@ static void decode_token(u16 token)
 		decode_default();
 		break;
 
-#if  0  /*  Fooey on C's petty limitations!  */
-        /*  I'd like to be able to do this:  */
-	/*  Special Functions  */
+#if  0				/*  Fooey on C's petty limitations!  */
```

Comment:
maybe start the comment on the next line so that the indenter knows what to do about it?